### PR TITLE
Add user methods: update user data, and get email

### PR DESCRIPTION
**Pull request by llunaCreixent**
- [x] Create methods: updateUser, and getEmail
- [x] Create tests
- [x] Use testing version of dockers
- [x] Add getEmail method
- [ ] Set dockers back to main in GHActions workflow file 

You can test with the `circles-docker` branch `update-users-feature`.

Closes https://github.com/CirclesUBI/circles-core/issues/123

### CODE REVIEW

**Comment by llunaCreixent** [comment]
Tests pass locally for me

**Comment by llunaCreixent** [comment]
@mikozet , I've published a testing version, so you can import it to test myxogastria.
Remember to use the new branches for the docker repo, and install the new core package in the myxo repo.

**Comment by mikozet** [comment]
Hi @llunaCreixent 
I run tests few times and it looks like they are failing at one point:
Zrzut ekranu 2022-07-28 o 09 24 29

**Comment by llunaCreixent** [comment]
> Hi @llunaCreixent 
> I run tests few times and it looks like they are failing at one point:
> Zrzut ekranu 2022-07-28 o 09 24 29
> 
> 

Then we have to reimplement this test somehow

**Review by JacqueGM — APPROVED** [review]
LGTM when version and branches have been updated properly ^^

File: src/user.js [hunk 1]
```diff
@@ -194,5 +254,58 @@ export default function createUserModule(web3, contracts, utils) {
...
+     * @namespace core.user.getEmail
+     *
+     * @param {Object} account - web3 account instance
+     * @param {Object} userOptions - options
+     * @param {string} userOptions.safeAddress - owned Safe address
+     * @param {string} userOptions.username - alphanumerical username
+     *
+     * @return {email} - Email of the user
+     */
+    getEmail: async (account, userOptions) => {
```
Comment (llunaCreixent) [inline src/user.js:270]: @mikozet, this is the method for getting the email

**Review by mikozet — APPROVED** [review]

**Pull request merged**